alpha/lib/model/LiveEntryServerNode.php

```diff
@@ -3,13 +3,57 @@
  * @package Core
  * @subpackage model
  */
-class LiveEntryServerNode extends EntryServerNode{
-
+class LiveEntryServerNode extends EntryServerNode
+{
 	const OM_CLASS = 'LiveEntryServerNode';
 	
 	const CUSTOM_DATA_STREAMS = "streams";
 	const CUSTOM_DATA_APPLICATION_NAME = "application_name";
 	const CUSTOM_DATA_DC = "dc";
+	
+	private $statusChanged = false;
+	
+	/* (non-PHPdoc)
+	 * @see BaseEntryServerNode::postSave()
+	 */
+	public function postSave(PropelPDO $con = null)
+	{
+		if($this->statusChanged)
```

Comment:
What's the purpose of the statusChanged flag? You can simply check the modified columns.
Also, if entry not found, log it as an error.
Also, use shorter texts in the logs.